changelog/23457.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+cli/snapshot: Add CLI tool to inspect Vault snapshots
```

Comment:
@elliesterner next time please use the correct new feature formatting for new features in the changelog.

Reply:
To add some more clarification - this should use the "Feature Name" we'd use in the release notes, and should try to communicate to the user why they would want to learn more about this brand new feature we're introducing in Vault 1.16.

Reply:
Thanks @mladlow . I thought I had corrected this [here](https://github.com/hashicorp/vault/pull/25401)?

Reply:
@elliesterner I see the PR, thanks! I think the correction didn't get merged to the 1.16 release branch, so on the 1.16 release branch it still has the original text. It's still being generated with this text. I added the backport label to the PR you linked. Could you please help ensure the 1.16 backport PR merges?

Reply:
oops, yes will fix. Thank you!